index.js

```diff
@@ -27,7 +39,70 @@ app.use(express.json());
 // IDEAS METHODS
 
 app.get('/ideas', (req, res) => {
-    res.status(200).json(getIdeas());
+    try {
+        if (req.query.sortBy === 'totalDonationSum') {
+            res.json(sortedByDonationSumIdeas());
+            return;
+        }
+        if (req.query.sortBy === 'status' && req.query.status === 'accepted') {
+            res.json(approvedIdeasList());
+            return;
+        }
+        if (req.query.sortBy === 'status' && req.query.status === 'pending') {
+            res.json(pendingIdeasList());
+            return;
+        }
+        if (req.query.sortBy === 'status' && req.query.status === 'rejected') {
+            res.json(rejectedIdeasList());
+            return;
+        }
+        res.json(getIdeas());
```

Comment:
Try to call `res.json` only once in this function.

Reply:
I've tryed ) 
Updated. Tested. Praying.